spring-integration-core/src/main/java/org/springframework/integration/support/management/DefaultMessageChannelMetrics.java

```diff
@@ -118,8 +119,9 @@ public DefaultMessageChannelMetrics(String name, ExponentialMovingAverage sendDu
 	 */
 	public DefaultMessageChannelMetrics(String name, ExponentialMovingAverage sendDuration,
 			ExponentialMovingAverageRate sendErrorRate, ExponentialMovingAverageRatio sendSuccessRatio,
-			ExponentialMovingAverageRate sendRate, Timer timer, Counter errorCounter) {
-		super(name, timer, errorCounter);
+			ExponentialMovingAverageRate sendRate, Timer timer, Counter errorCounter, Counter receiveCounter,
+			Counter receiveErrorCounter) {
+		super(name, timer, errorCounter, receiveCounter, receiveErrorCounter);
```

Comment:
Shound't we use getters then for these `errorCounter, receiveCounter, receiveErrorCounter` in the code below?